LICENSE

```diff
@@ -1,21 +1,21 @@
-    MIT License
-

```

Comment:
I didn't, when I cloned the repo it converted the CR's to CR LF. I think the license was originally created on a Mac because that are the only OS that ends with CR instead of LF or CR LF. VS Code does not give me the option to revert back to the CR on LF or CR LF.